Changing the cluster pg and glossary (#2796)
temporalio/documentation: docs/encyclopedia/clusters.mdx

  - term
---

> :::info
> Please note an important update in our terminology. We now refer to the Temporal Cluster as the Temporal Service.
> :::

Reviewer: ```suggestion
:::info

Please note an important update in our terminology. 
We now refer to a "Temporal Cluster" as a "Temporal Service".

:::
```

> This guide provides a comprehensive overview of the Temporal Service.

Reviewer: ```suggestion
This page provides a comprehensive technical overview of a Temporal Service.
```

A Temporal Service is the group of services, known as the [Temporal Server](#temporal-server), combined with [Persistence](#persistence) and [Visibility](#visibility) stores, that together act as a component of the Temporal Platform.

> - [Temporal Service deployment guide](/self-hosted-guide)

Reviewer: ```suggestion
- See the [Self-hosted Temporal Service production deployment guide](/self-hosted-guide) for implementation guidance.
```
